### PR TITLE
Add pytest-3 compatibility

### PR DESCRIPTION
**Pull request by vkondula**
fixes: #46

### CODE REVIEW

**Comment by vkondula** [comment]
@lukas-bednar  how I am suppose to pass codecov? Run everything with both pytest-2 and pytest-3?

File: requirements-pytest29.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+pytest>=2.2.4,<3.0.0
```
Comment (lukas-bednar) [inline requirements-pytest29.txt]: I would call this file just requirements-pytest2.txt

File: tox.ini [hunk 2]
```diff
@@ -2,9 +2,10 @@
 envlist = py26,py27,py34,py35,pep8
 [tox:travis]
 2.6 = py26
-2.7 = py27, pep8
+2.7 = py27, pep8, py27-pytest29
```
Comment (lukas-bednar) [inline tox.ini]: py27-pytest2

File: tox.ini [hunk 3]
```diff
@@ -19,3 +20,16 @@ commands=
       {posargs}
 [testenv:pep8]
 commands=flake8 setup.py pytest_jira.py tests
+[testenv:py27-pytest29]
```
Comment (lukas-bednar) [inline tox.ini]: testenv:py27-pytest2

File: tox.ini [hunk 4]
```diff
@@ -19,3 +20,16 @@ commands=
...
+deps=
+  -rrequirements-test.txt
+  -rrequirements-pytest29.txt
```
Comment (lukas-bednar) [inline tox.ini]: ``-rrequirements-pytest2.txt``

**Pull request merged**